src/plugins/cosmos/components/modals/Staked/Staked.tsx

```diff
@@ -0,0 +1,38 @@
+import { Modal, ModalContent, ModalOverlay } from '@chakra-ui/react'
+import React, { useRef } from 'react'
+import { MemoryRouter, Route, RouteComponentProps, Switch } from 'react-router-dom'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+
+import { Staked } from './views/Staked'
+
+type StakedModalProps = {
+  assetId: string
+}
+
+export enum StakedModalRoutes {
+  Staked = '/defi/modal/staked'
```

Comment:
Note: Since these are virtual routes, naming doesn't really matter here as it won't be shown to user. I'd like to take some time to rename all the routes to `plugins/<cosmosCaip19/modal/<modalName>` in a follow-up PR however.